GravatarApp/Logging/GravatarCrashLoggingProvider.swift

```diff
@@ -0,0 +1,39 @@
+import AutomatticRemoteLogging
+
+actor GravatarCrashLogger {
+    static let shared = GravatarCrashLogger()
+    private let crashLogging: CrashLogging
+
+    private init() {
+        self.crashLogging = CrashLogging(
+            dataProvider: GravatarCrashLoggingDataProvider()
+        )
+    }
+
+    func start() {
+        do {
+            _ = try crashLogging.start()
+        } catch {
+            print("⚠️ Crash logging failed to start: \(error)")
+        }
+    }
+}
+
+struct GravatarCrashLoggingDataProvider: CrashLoggingDataProvider {
+    let sentryDSN = Secrets.sentryDSN
+    let userHasOptedOut = false
+    let shouldEnableAutomaticSessionTracking = true
```

Comment:
FYI: We may want to set this to the default of `false` until opt-out functionality is implemented.